File: nodejs/helloworld.js
var startTime = new Date().getTime();

const http = require('http');

const requestListener = function (req, res) {
  res.writeHead(200);
  res.end('Hello, World!');
};

const server = http.createServer(requestListener);
server.listen(8080);

var endTime = new Date().getTime();

console.log('TIME1: ' + startTime);
console.log('TIME2: ' + endTime);

File: go/src/helloworld.go
package main

import (
    "fmt"
    "net/http"
    "time"
)

type HttpHandler struct{}

func (h HttpHandler) ServeHTTP(res http.ResponseWriter, req *http.Request) {
    data := []byte("Hello World!")
    res.Write(data)
}

func main() {
    time1 := time.Now()
    fmt.Printf("TIME1: %d\n", time1.UnixMilli())
    handler := HttpHandler{}
    time2 := time.Now()
    fmt.Printf("TIME2: %d\n", time2.UnixMilli())
    http.ListenAndServe(":8080", handler)
}
